http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -104,6 +110,10 @@ docker run -d --restart always --network host --name listener \
 --env LISTENER_RELAYCONNECTIONNAME=$RELAY_CONNECTION_NAME \
 --env LISTENER_SAMINSPECTORPROPERTIES_SAMRESOURCEID=$SAMRESOURCEID \
 --env LISTENER_SAMINSPECTORPROPERTIES_SAMURL=$SAMURL \
+--env LISTENER_SETDATEACCESSEDINSPECTORPROPERTIES_SERVICEHOST=$LEONARDOURL \
```

Comment:
I think you also need to enable the `setDateAccessed` inspected as well -- by default only the [samChecked](https://github.com/DataBiosphere/terra-azure-relay-listeners/blob/main/service/src/main/resources/application.yml#L20) is enabled.

My syntax could be off but something like:
```
--env LISTENER_REQUESTINSPECTORS_0=samChecker \
--env LISTENER_REQUESTINSPECTORS_1=setDateAccessed
```